test/e2e/events_test.go

```diff
@@ -199,4 +199,16 @@ var _ = Describe("Podman events", func() {
 
 		wg.Wait()
 	})
+
+	It("podman events pod creation", func() {
+		SkipIfNotFedora()
```

Comment:
Why?

Reply:
I was following the other tests, I will try without this and see what happens